osu.Game/Overlays/Toolbar/TransientUserStatisticsUpdateDisplay.cs

```diff
@@ -0,0 +1,224 @@
+                    Children = new Drawable[]
+                    {
+                        mainValue = new Counter<T>
+                        {
+                            ValuePrefix = mainValuePrefix,
+                            Anchor = Anchor.CentreRight,
+                            Origin = Anchor.CentreRight,
+                            Font = OsuFont.GetFont(),
+                        },
+                        new Container
+                        {
+                            AutoSizeAxes = Axes.Both,
+                            Anchor = Anchor.CentreRight,
+                            Origin = Anchor.CentreRight,
+                            Children = new Drawable[]
+                            {
+                                deltaValue = new Counter<T>
+                                {
+                                    Anchor = Anchor.CentreRight,
+                                    Origin = Anchor.CentreRight,
+                                    Font = OsuFont.GetFont(size: 12, fixedWidth: false, weight: FontWeight.SemiBold),
+                                    AlwaysPresent = true,
+                                },
+                                titleText = new OsuSpriteText
+                                {
+                                    Anchor = Anchor.CentreRight,
+                                    Origin = Anchor.CentreRight,
+                                    Font = OsuFont.Default.With(size: 12, weight: FontWeight.SemiBold),
+                                    Text = title,
+                                    AlwaysPresent = true,
+                                }
+                            }
+                        }
+                    }
+                };
+            }
+
+            public void Display(T before, T delta, T after)
+            {
+                int comparison = valueComparer.Compare(before, after);
+
+                if (comparison > 0)
+                {
+                    deltaValue.Colour = colours.Lime1;
+                    deltaValue.ValuePrefix = "+";
+                }
+                else if (comparison < 0)
+                {
+                    deltaValue.Colour = colours.Red1;
+                    deltaValue.ValuePrefix = "-";
+                }
+                else
+                {
+                    deltaValue.Colour = Colour4.White;
+                    deltaValue.ValuePrefix = string.Empty;
+                }
+
+                mainValue.SetCountWithoutRolling(before);
+                deltaValue.SetCountWithoutRolling(delta);
+
+                titleText.Alpha = 1;
+                deltaValue.Alpha = 0;
+
+                using (BeginDelayedSequence(1000))
+                {
+                    titleText.FadeOut(250, Easing.OutQuint);
+                    deltaValue.FadeIn(250, Easing.OutQuint)
+                              .Then().Delay(1000)
+                              .Then().OnComplete(_ =>
```

Comment:
The delay here seems to be broken. The delta value just rolls down immediately and it's hard to know the initial number.

Reply:
fixed in https://github.com/ppy/osu/pull/27156/commits/6baa0999065adc10f3b61ee527fecc614f6fa1f1, i think